yellowbrick/features/importances.py

```diff
@@ -150,8 +151,11 @@ def fit(self, X, y=None, **kwargs):
         # therefore we flatten by taking the average by
         # column to get shape (n_features,)  (see LogisticRegression)
         if not self.stack and self.feature_importances_.ndim > 1:
-            self.feature_importances_ = np.mean(self.feature_importances_,
-                                                axis=0)
+            self.feature_importances_ = np.mean(self.feature_importances_, axis=0)
+            warnings.warn((
+                "detected multi-dimensional feature importances but stack=False, "
+                "using mean to aggregate them."
+            ), YellowbrickWarning)
```

Comment:
This warning is issued because it is likely the user desires stack=True and should set it to avoid misinterpreting the feature importances.